README.md

```diff
@@ -74,29 +74,33 @@ err := flow.Run(ctx, input, &output)
 ## Architecture Overview
 
 ```mermaid
+---
+config:
```

Comment:
`--- config: layout: elk ---` → `%%{ init: { "layout": "elk" } }%%`
Using `%%{ init: ... }%%` is Mermaid’s official config syntax.